src/launcher/index.ts

```diff
@@ -56,6 +56,38 @@ async function main() {
     log.error(error)
   })
 
+  const openBrowserWindow = async (
+    flowrStore: Store<IFlowrStore>,
```

Comment:
Shadowed variable name

Reply:
`flowrStore` is declared only after this function in the `onReady`